app/models/service_update.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class ServiceUpdate
+  CLAIMS_SERVICE_UPDATES_YAML_FILE =
+    Rails.root.join("db/claims_service_updates.yml").freeze
+  PLACEMENTS_SERVICE_UPDATES_YAML_FILE =
+    Rails.root.join("db/placements_service_updates.yml").freeze
+
+  include ActiveModel::Model
+
+  attr_accessor :date, :title, :content
+
+  def id
+    title.parameterize
+  end
+
+  def self.where(service:)
+    path = yaml_file(service:)
+    updates = YAML.load_file(path) if path
+
+    return {} unless updates
+
+    updates.map { |params| new(params) }
+  end
+
+  def self.yaml_file(service:)
+    case service
+    when :claims
+      Rails.root.join(CLAIMS_SERVICE_UPDATES_YAML_FILE)
```

Comment:
Is the `Rails.root.join` useful here? Seems like `CLAIMS_SERVICE_UPDATES_YAML_FILE` already used it

Reply:
Agreed.